models/varnet/varnet.py

```diff
@@ -58,13 +58,28 @@ def __call__(self, kspace, target, attrs, fname, slice):
                 mean (float): Mean value used for normalization.
                 std (float): Standard deviation value used for normalization.
         """
-        target = T.to_tensor(target)
+        if target is not None:
+            target = T.to_tensor(target)
+            max_value = attrs['max']
+        else:
+            target = torch.zeros([self.resolution,self.resolution],dtype=torch.float32)
```

Comment:
These target and max_value isn't ever used, right? 

Can we just have:
target = None
max_value = None

Or if the dataloader collate function doesn't work with None (can't remember if it does):
target = torch.tensor(0)
max_value = 0.0

Reply:
I found that the dataloader collate function was throwing errors with None so I just assigned 0 values.

Reply:
New commit should have 
target = torch.tensor(0)
max_value = 0.0